certified-connectors/ZohoSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1671 @@
+                            "type": "integer",
+                            "format": "int32",
+                            "description": "Signing order"
+                          },
+                          "recipient_name": {
+                            "type": "string",
+                            "description": "Recipient name"
+                          },
+                          "action_status": {
+                            "type": "string",
+                            "description": "Action status"
+                          },
+                          "recipient_countrycode": {
+                            "type": "string",
+                            "description": "Recipient countrycode"
+                          },
+                          "ishost": {
+                            "type": "boolean",
+                            "description": "host usage status"
+                          },
+                          "language": {
+                            "type": "string",
+                            "description": "language"
+                          },
+                          "in_person_name": {
+                            "type": "string",
+                            "description": "In person name"
+                          },
+                          "in_person_email": {
+                            "type": "string",
+                            "description": "In person email"
+                          },
+                          "verification_code": {
+                            "type": "string",
+                            "description": "Verification code"
+                          }
+                        }
+                      },
+                      "description": "actions"
+                    },
+                    "attachment_size": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Size of the attachment"
+                    }
+                  },
+                  "description": "requests"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get a document",
+        "description": "Get details of a particular document",
+        "operationId": "get-document",
+        "parameters": [
+          {
+            "name": "RequestId",
+            "in": "path",
+            "required": true,
+            "type": "integer",
+            "format": "int64",
+            "x-ms-url-encoding": "single",
+            "description": "Request ID",
+            "x-ms-summary": "Request ID"
+          }
+        ]
+      },
+      "put": {
+        "responses": {
+          "default": {
```

Comment:
default responses are not allowed, please provide a more specific response code. Thank you